src/singletons/Toasts.hpp

```diff
@@ -7,17 +7,29 @@ namespace chatterino {
 
 enum class Platform : uint8_t;
 
+enum class ToastReactions {
```

Comment:
Should be called `ToastReaction` (singular) not `ToastReactions` (plural)

Reply:
done